deploy/scripts/TLSSettings.ps1

```diff
@@ -0,0 +1,738 @@
+﻿#***************************************************************************************************************
```

Comment:
Is this script subject to any copyright/licensing model? If so, we should add it to our thirdparty notices here: https://raw.githubusercontent.com/Azure/Industrial-IoT/main/thirdpartynotices.txt . If not, let's add our MIT header.

Reply:
This is not subject to any copyright/licensing model. Our scripts does not have any MIT header. Resolving the comment.